streaming-mqtt/README.md

```diff
@@ -52,12 +52,14 @@ this actor can be configured to handle failures, etc.
 
     val lines = MQTTUtils.createStream(ssc, brokerUrl, topic)
     val lines = MQTTUtils.createPairedStream(ssc, brokerUrl, topic)
+    val lines = MQTTUtils.createPairedByteArrayStreamStream(ssc, brokerUrl, topic)
```

Comment:
@davidrosenstark -- I assume the `StreamStream` word duplication is a copy-paste error?

~`val lines = MQTTUtils.createPairedByteArrayStreamStream(ssc, brokerUrl, topic)`~
`val lines = MQTTUtils.createPairedByteArrayStream(ssc, brokerUrl, topic)`